### PR TITLE
Use new minio buckets

### PR DESCRIPTION
**Pull request by redhug1**
### What

As title

### How to review

Visual inspect

### Who can review

Anyone

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -68,15 +69,17 @@ func Get() (*Config, error) {
...
-		MinioAccessKey:             "",
-		MinioSecretKey:             "",
-		VaultPath:                  "secret/shared/psk",
-		VaultAddress:               "http://localhost:8200",
-		VaultToken:                 "",
-		ComponentTestUseLogFile:    false,
-		EncryptionDisabled:         false, // needed for local development to skip needing vault - TODO - remove if not needed
+		UploadBucketName:           "public-bucket", // where to place the created .xlsx
+		PrivateUploadBucketName:    "private-bucket",
+		//		UploadBucketName:           "dp-cantabular-csv-exporter", // where to place the created .xlsx
```
Comment (DavidSubiros) [inline config/config.go:74]: I think you can remove this comment (UploadBucketName is defined above) - maybe in the next PR.

File: features/compose/minio.yml [hunk 2]
```diff
@@ -7,5 +7,6 @@ services:
     environment:
       - MINIO_ACCESS_KEY=minio-access-key
       - MINIO_SECRET_KEY=minio-secret-key
-      - MINIO_DEFAULT_BUCKETS=dp-cantabular-csv-exporter
-      
\ No newline at end of file
+#      - MINIO_DEFAULT_BUCKETS=dp-cantabular-csv-exporter
```
Comment (DavidSubiros) [inline features/compose/minio.yml:10]: this line can also be removed (in another pr)

**Review by DavidSubiros — APPROVED** [review]

File: config/config.go (hunk 1 above)
Comment (redhug1) [reply config/config.go:74]: Yes, thats in the next PR.

File: features/compose/minio.yml (hunk 2 above)
Comment (redhug1) [reply features/compose/minio.yml:10]: Yes, thats in the next PR.

**Pull request merged**